INFO:tensorflow:Using default config.
WARNING:tensorflow:Using temporary folder as model directory: /tmp/tmpgq0k9jdy
INFO:tensorflow:Using config: {'_model_dir': '/tmp/tmpgq0k9jdy', '_tf_random_seed': None, '_save_summary_steps': 100, '_save_checkpoints_steps': None, '_save_checkpoints_secs': 600, '_session_config': allow_soft_placement: true
graph_options {
  rewrite_options {
    meta_optimizer_iterations: ONE
  }
}
, '_keep_checkpoint_max': 5, '_keep_checkpoint_every_n_hours': 10000, '_log_step_count_steps': 100, '_train_distribute': None, '_device_fn': None, '_protocol': None, '_eval_distribute': None, '_experimental_distribute': None, '_service': None, '_cluster_spec': <tensorflow.python.training.server_lib.ClusterSpec object at 0x7f53694a5e10>, '_task_type': 'worker', '_task_id': 0, '_global_id_in_cluster': 0, '_master': '', '_evaluation_master': '', '_is_chief': True, '_num_ps_replicas': 0, '_num_worker_replicas': 1}
WARNING:tensorflow:From /home/barreira/Privacy/env/lib/python3.7/site-packages/tensorflow/python/framework/op_def_library.py:263: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
WARNING:tensorflow:From /home/barreira/Privacy/env/lib/python3.7/site-packages/tensorflow_estimator/python/estimator/inputs/queues/feeding_queue_runner.py:62: QueueRunner.__init__ (from tensorflow.python.training.queue_runner_impl) is deprecated and will be removed in a future version.
Instructions for updating:
To construct input pipelines, use the `tf.data` module.
WARNING:tensorflow:From /home/barreira/Privacy/env/lib/python3.7/site-packages/tensorflow_estimator/python/estimator/inputs/queues/feeding_functions.py:500: add_queue_runner (from tensorflow.python.training.queue_runner_impl) is deprecated and will be removed in a future version.
Instructions for updating:
To construct input pipelines, use the `tf.data` module.
INFO:tensorflow:Calling model_fn.
WARNING:tensorflow:From /home/barreira/Privacy/env/lib/python3.7/site-packages/tensorflow/python/keras/layers/core.py:143: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
WARNING:tensorflow:From /home/barreira/Privacy/env/lib/python3.7/site-packages/tensorflow/python/ops/array_grad.py:425: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
/home/barreira/Privacy/env/lib/python3.7/site-packages/tensorflow/python/ops/gradients_impl.py:110: UserWarning: Converting sparse IndexedSlices to a dense Tensor of unknown shape. This may consume a large amount of memory.
  "Converting sparse IndexedSlices to a dense Tensor of unknown shape. "
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Create CheckpointSaverHook.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Graph was finalized.
2019-06-11 16:35:59.429881: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
2019-06-11 16:35:59.434904: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2303580000 Hz
2019-06-11 16:35:59.435140: I tensorflow/compiler/xla/service/service.cc:150] XLA service 0x55d76b328ea0 executing computations on platform Host. Devices:
2019-06-11 16:35:59.435164: I tensorflow/compiler/xla/service/service.cc:158]   StreamExecutor device (0): <undefined>, <undefined>
OMP: Info #212: KMP_AFFINITY: decoding x2APIC ids.
OMP: Info #210: KMP_AFFINITY: Affinity capable, using global cpuid leaf 11 info
OMP: Info #154: KMP_AFFINITY: Initial OS proc set respected: 0,1
OMP: Info #156: KMP_AFFINITY: 2 available OS procs
OMP: Info #157: KMP_AFFINITY: Uniform topology
OMP: Info #179: KMP_AFFINITY: 2 packages x 1 cores/pkg x 1 threads/core (2 total cores)
OMP: Info #214: KMP_AFFINITY: OS proc to physical thread map:
OMP: Info #171: KMP_AFFINITY: OS proc 0 maps to package 0 
OMP: Info #171: KMP_AFFINITY: OS proc 1 maps to package 2 
OMP: Info #250: KMP_AFFINITY: pid 43238 tid 43238 thread 0 bound to OS proc set 0
2019-06-11 16:35:59.443823: I tensorflow/core/common_runtime/process_util.cc:71] Creating new thread pool with default inter op setting: 2. Tune using inter_op_parallelism_threads for best performance.
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
WARNING:tensorflow:From /home/barreira/Privacy/env/lib/python3.7/site-packages/tensorflow/python/training/monitored_session.py:809: start_queue_runners (from tensorflow.python.training.queue_runner_impl) is deprecated and will be removed in a future version.
Instructions for updating:
To construct input pipelines, use the `tf.data` module.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Saving checkpoints for 0 into /tmp/tmpgq0k9jdy/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
OMP: Info #250: KMP_AFFINITY: pid 43238 tid 43256 thread 1 bound to OS proc set 1
OMP: Info #250: KMP_AFFINITY: pid 43238 tid 43290 thread 2 bound to OS proc set 0
OMP: Info #250: KMP_AFFINITY: pid 43238 tid 43257 thread 3 bound to OS proc set 1
OMP: Info #250: KMP_AFFINITY: pid 43238 tid 43291 thread 4 bound to OS proc set 0
INFO:tensorflow:loss = 2.3147297, step = 1
INFO:tensorflow:Saving checkpoints for 58 into /tmp/tmpgq0k9jdy/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Loss for final step: 2.0750031.
INFO:tensorflow:Calling model_fn.
WARNING:tensorflow:From /home/barreira/Privacy/env/lib/python3.7/site-packages/tensorflow/python/ops/metrics_impl.py:455: to_float (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Starting evaluation at 2019-06-11T15:37:12Z
INFO:tensorflow:Graph was finalized.
WARNING:tensorflow:From /home/barreira/Privacy/env/lib/python3.7/site-packages/tensorflow/python/training/saver.py:1266: checkpoint_exists (from tensorflow.python.training.checkpoint_management) is deprecated and will be removed in a future version.
Instructions for updating:
Use standard file APIs to check for files with this prefix.
INFO:tensorflow:Restoring parameters from /tmp/tmpgq0k9jdy/model.ckpt-58
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Finished evaluation at 2019-06-11-15:37:12
INFO:tensorflow:Saving dict for global step 58: accuracy = 0.461, global_step = 58, loss = 2.06569
INFO:tensorflow:Saving 'checkpoint_path' summary for global step 58: /tmp/tmpgq0k9jdy/model.ckpt-58
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Create CheckpointSaverHook.
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmpgq0k9jdy/model.ckpt-58
WARNING:tensorflow:From /home/barreira/Privacy/env/lib/python3.7/site-packages/tensorflow/python/training/saver.py:1070: get_checkpoint_mtimes (from tensorflow.python.training.checkpoint_management) is deprecated and will be removed in a future version.
Instructions for updating:
Use standard file utilities to get mtimes.
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Saving checkpoints for 58 into /tmp/tmpgq0k9jdy/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:loss = 2.0639415, step = 59
INFO:tensorflow:Saving checkpoints for 116 into /tmp/tmpgq0k9jdy/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Loss for final step: 1.8494968.
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Starting evaluation at 2019-06-11T15:38:28Z
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmpgq0k9jdy/model.ckpt-116
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Finished evaluation at 2019-06-11-15:38:28
INFO:tensorflow:Saving dict for global step 116: accuracy = 0.7105, global_step = 116, loss = 1.8439268
INFO:tensorflow:Saving 'checkpoint_path' summary for global step 116: /tmp/tmpgq0k9jdy/model.ckpt-116
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Create CheckpointSaverHook.
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmpgq0k9jdy/model.ckpt-116
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Saving checkpoints for 116 into /tmp/tmpgq0k9jdy/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:loss = 1.8352385, step = 117
INFO:tensorflow:Saving checkpoints for 174 into /tmp/tmpgq0k9jdy/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Loss for final step: 1.7411189.
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Starting evaluation at 2019-06-11T15:39:50Z
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmpgq0k9jdy/model.ckpt-174
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Finished evaluation at 2019-06-11-15:39:51
INFO:tensorflow:Saving dict for global step 174: accuracy = 0.8021, global_step = 174, loss = 1.7383394
INFO:tensorflow:Saving 'checkpoint_path' summary for global step 174: /tmp/tmpgq0k9jdy/model.ckpt-174
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Create CheckpointSaverHook.
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmpgq0k9jdy/model.ckpt-174
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Saving checkpoints for 174 into /tmp/tmpgq0k9jdy/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:loss = 1.7703667, step = 175
INFO:tensorflow:Saving checkpoints for 232 into /tmp/tmpgq0k9jdy/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Loss for final step: 1.7221775.
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Starting evaluation at 2019-06-11T15:41:22Z
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmpgq0k9jdy/model.ckpt-232
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Finished evaluation at 2019-06-11-15:41:22
INFO:tensorflow:Saving dict for global step 232: accuracy = 0.8172, global_step = 232, loss = 1.7001264
INFO:tensorflow:Saving 'checkpoint_path' summary for global step 232: /tmp/tmpgq0k9jdy/model.ckpt-232
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Create CheckpointSaverHook.
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmpgq0k9jdy/model.ckpt-232
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Saving checkpoints for 232 into /tmp/tmpgq0k9jdy/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:loss = 1.672147, step = 233
INFO:tensorflow:Saving checkpoints for 290 into /tmp/tmpgq0k9jdy/model.ckpt.
WARNING:tensorflow:From /home/barreira/Privacy/env/lib/python3.7/site-packages/tensorflow/python/training/saver.py:966: remove_checkpoint (from tensorflow.python.training.checkpoint_management) is deprecated and will be removed in a future version.
Instructions for updating:
Use standard file APIs to delete files with this prefix.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Loss for final step: 1.6564922.
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Starting evaluation at 2019-06-11T15:42:55Z
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmpgq0k9jdy/model.ckpt-290
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Finished evaluation at 2019-06-11-15:42:55
INFO:tensorflow:Saving dict for global step 290: accuracy = 0.8246, global_step = 290, loss = 1.6799406
INFO:tensorflow:Saving 'checkpoint_path' summary for global step 290: /tmp/tmpgq0k9jdy/model.ckpt-290
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Create CheckpointSaverHook.
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmpgq0k9jdy/model.ckpt-290
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Saving checkpoints for 290 into /tmp/tmpgq0k9jdy/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:loss = 1.6403161, step = 291
INFO:tensorflow:Saving checkpoints for 348 into /tmp/tmpgq0k9jdy/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Loss for final step: 1.666453.
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Starting evaluation at 2019-06-11T15:44:32Z
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmpgq0k9jdy/model.ckpt-348
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Finished evaluation at 2019-06-11-15:44:33
INFO:tensorflow:Saving dict for global step 348: accuracy = 0.83, global_step = 348, loss = 1.6675116
INFO:tensorflow:Saving 'checkpoint_path' summary for global step 348: /tmp/tmpgq0k9jdy/model.ckpt-348
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Create CheckpointSaverHook.
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmpgq0k9jdy/model.ckpt-348
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Saving checkpoints for 348 into /tmp/tmpgq0k9jdy/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:loss = 1.6555243, step = 349
INFO:tensorflow:Saving checkpoints for 406 into /tmp/tmpgq0k9jdy/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Loss for final step: 1.62643.
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Starting evaluation at 2019-06-11T15:46:14Z
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmpgq0k9jdy/model.ckpt-406
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Finished evaluation at 2019-06-11-15:46:14
INFO:tensorflow:Saving dict for global step 406: accuracy = 0.8328, global_step = 406, loss = 1.6587287
INFO:tensorflow:Saving 'checkpoint_path' summary for global step 406: /tmp/tmpgq0k9jdy/model.ckpt-406
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Create CheckpointSaverHook.
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmpgq0k9jdy/model.ckpt-406
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Saving checkpoints for 406 into /tmp/tmpgq0k9jdy/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:loss = 1.6924553, step = 407
INFO:tensorflow:Saving checkpoints for 464 into /tmp/tmpgq0k9jdy/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Loss for final step: 1.6493248.
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Starting evaluation at 2019-06-11T15:48:01Z
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmpgq0k9jdy/model.ckpt-464
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Finished evaluation at 2019-06-11-15:48:02
INFO:tensorflow:Saving dict for global step 464: accuracy = 0.8361, global_step = 464, loss = 1.6521881
INFO:tensorflow:Saving 'checkpoint_path' summary for global step 464: /tmp/tmpgq0k9jdy/model.ckpt-464
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Create CheckpointSaverHook.
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmpgq0k9jdy/model.ckpt-464
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Saving checkpoints for 464 into /tmp/tmpgq0k9jdy/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:loss = 1.6730131, step = 465
INFO:tensorflow:Saving checkpoints for 522 into /tmp/tmpgq0k9jdy/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Loss for final step: 1.6404415.
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Starting evaluation at 2019-06-11T15:49:40Z
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmpgq0k9jdy/model.ckpt-522
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Finished evaluation at 2019-06-11-15:49:41
INFO:tensorflow:Saving dict for global step 522: accuracy = 0.8388, global_step = 522, loss = 1.6464282
INFO:tensorflow:Saving 'checkpoint_path' summary for global step 522: /tmp/tmpgq0k9jdy/model.ckpt-522
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Create CheckpointSaverHook.
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmpgq0k9jdy/model.ckpt-522
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Saving checkpoints for 522 into /tmp/tmpgq0k9jdy/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:loss = 1.6503532, step = 523
INFO:tensorflow:Saving checkpoints for 580 into /tmp/tmpgq0k9jdy/model.ckpt.
WARNING:tensorflow:Issue encountered when serializing critical_section_executions.
Type is unsupported, or the types of the items don't match field type in CollectionDef. Note this is a warning and probably safe to ignore.
'_ExecutionSignature' object has no attribute 'name'
INFO:tensorflow:Loss for final step: 1.6256702.
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Starting evaluation at 2019-06-11T15:51:26Z
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from /tmp/tmpgq0k9jdy/model.ckpt-580
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Finished evaluation at 2019-06-11-15:51:27
INFO:tensorflow:Saving dict for global step 580: accuracy = 0.8698, global_step = 580, loss = 1.6246982
INFO:tensorflow:Saving 'checkpoint_path' summary for global step 580: /tmp/tmpgq0k9jdy/model.ckpt-580

WARNING: The TensorFlow contrib module will not be included in TensorFlow 2.0.
For more information, please see:
  * https://github.com/tensorflow/community/blob/master/rfcs/20180907-contrib-sunset.md
  * https://github.com/tensorflow/addons
If you depend on functionality not listed there, please file an issue.

For delta=1e-5, the current epsilon is: 1.56
Test accuracy after 1 epochs is: 0.461
For delta=1e-5, the current epsilon is: 1.73
Test accuracy after 2 epochs is: 0.711
For delta=1e-5, the current epsilon is: 1.89
Test accuracy after 3 epochs is: 0.802
For delta=1e-5, the current epsilon is: 2.03
Test accuracy after 4 epochs is: 0.817
For delta=1e-5, the current epsilon is: 2.17
Test accuracy after 5 epochs is: 0.825
For delta=1e-5, the current epsilon is: 2.30
Test accuracy after 6 epochs is: 0.830
For delta=1e-5, the current epsilon is: 2.42
Test accuracy after 7 epochs is: 0.833
For delta=1e-5, the current epsilon is: 2.54
Test accuracy after 8 epochs is: 0.836
For delta=1e-5, the current epsilon is: 2.66
Test accuracy after 9 epochs is: 0.839
For delta=1e-5, the current epsilon is: 2.77
Test accuracy after 10 epochs is: 0.870
Elapsed time:  928.5065948963165
